file_browser/src/components/AddDIalog.tsx

```diff
@@ -0,0 +1,106 @@
+import { Dispatch, SetStateAction, useState } from "react";
+import { SystemItem } from "../classes/SystemItem";
+import { FileClass } from "../classes/File";
+import { Folder } from "../classes/Folder";
+import { observer } from "mobx-react-lite";
+import fileSystemStore from "../stores/FileSystemStore";
+
+const options = ["File", "Folder"];
+
+function AddDialog({
+  setDialog,
+}: {
+  setDialog: Dispatch<SetStateAction<boolean>>;
+}) {
+  const [selected, setSelected] = useState<any>(0);
```

Comment:
This is obviously a number, why did you put `any` as the type?
Also, seems like the only possible values are `0` and `1`. In this case it's common to use boolean instead.